client/src/components/Overview.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
```

Comment:
So i think its best to not include these component files (Overview, Q&A, & Ratings) in the merge because they will for sure result in a merge conflict when one of use goes to merge our feature branches into staging.

Reply:
done!